can-compute_test.js

```diff
@@ -6,6 +6,7 @@ var QUnit = require('steal-qunit');
 var canBatch = require('can-event/batch/');
 var Observation = require('can-observation');
 var domDispatch = require("can-util/dom/dispatch/dispatch");
+var isEmptyObject = require("can-util/js/is-empty-object/is-empty-object")
```

Comment:
You should remove this so `jshint` stops complaining.

Reply:
Yea, I ran tests in the browser and forgot to run it in cl before pushing. I'll fix it.